.changeset/lemon-cheetahs-live.md

```diff
@@ -0,0 +1,21 @@
+---
+"@utrecht/design-tokens": major
+"@utrecht/alert-css": major
+"@utrecht/components": major
+"@utrecht/component-library-angular": major
+"@utrecht/component-library-css": major
+"@utrecht/component-library-react": major
+"@utrecht/component-library-vue": major
+"@utrecht/web-component-library-angular": major
+"@utrecht/web-component-library-react": major
+"@utrecht/web-component-library-stencil": major
+"@utrecht/web-component-library-vue": major
```

Comment:
@Robbert wat is hier precies de bedoeling? Ik ben er maar even vanuit gegaan dat deze ook handmatig gebumped moeten worden omdat ze de alert gebruiken.

Reply:
Je hebt 't precies goed gedaan! 💯